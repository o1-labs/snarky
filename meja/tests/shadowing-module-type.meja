module type M = {
  type t;
};

module type M = {
  type u;
};
