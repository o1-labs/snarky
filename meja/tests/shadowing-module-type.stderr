File "tests/shadowing-module-type.meja", line 5, characters 12-13:
Error: There is already a module type with name M.
